include/libzfs_core.h

```diff
@@ -137,6 +137,20 @@ int lzc_wait_fs(const char *, zfs_wait_activity_t, boolean_t *);
 
 int lzc_set_bootenv(const char *, const nvlist_t *);
 int lzc_get_bootenv(const char *, nvlist_t **);
+
+void* __do_send_output(void* voidargs);
+#ifdef __linux__
```

Comment:
I feel like this definitely shouldn't pollute the global namespace

Reply:
Seems reasonable. As above, I'll just land it in the top of libzfs_core.c - and if that seems unreasonable, the only other place I can see it going is inside the function using it.